docs/changelog.rst

```diff
@@ -14,6 +16,7 @@ CHANGELOG
 **New features**
 
 * Publication workflow (fixes #1018)
+* Allow to add links to Youtube or Soundcloud media as attachment
```

Comment:
I'm not sure we will provide integration code for both Youtube And Vimeo in Geotrek-Rando